install-build-deps.sh

```diff
@@ -24,7 +24,8 @@ if [ -e /usr/bin/apt-get ] ; then
   sudo apt-get -y install zsync git libarchive-dev autoconf libtool make gcc g++ libtool libfuse-dev \
   liblzma-dev libglib2.0-dev libssl-dev libinotifytools0-dev liblz4-dev libcairo-dev desktop-file-utils cmake
   # libtool-bin might be required in newer distributions but is not available in precise
-  sudo cp resources/liblz4.pc /usr/lib/$ARCH-linux-gnu/pkgconfig/
+  test -e /usr/lib/$ARCH-linux-gnu/pkgconfig/liblz4.pc ||
```

Comment:
Any reason to use `test`? How wide spread is it? Wouldn't `if [ ! -f ... ]` be more easy to read?

Reply:
'[ ... ]' is actually shell suggar around TEST(1POSIX), and since test is Posix and used by the uber-portable autoconf, it's good enough to be used by mere mortals ;-)
I.e. it doesn't really matter, I just default to using 'test' over '['.